coralproject/talk
# [CORL-1176] YouTube and Twitter frame-ancestors bug

## What does this PR do?

Adds `'self'` to the `frame-ancestors` directive on the `Content-Security-Policy` header to permit oEmbed's.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Load a YouTube/Twitter preview and see it appear when the site origin is not the origin of the Coral instance.
